### ISSUE TITLE
twisted.conch.ssh.userauth should be ported to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8654]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)| @einarfd reported|
|-|-|
|Trac ID|trac#8654|
|Type|enhancement|
|Created|2016-07-22 11:40:06Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8654 8654
type__enhancement enhancement
reporter__einarfd einarfd
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469187606323431 1469187606323431
changetime__1474198049187875 1474198049187875
version__None None
owner__einarfd einarfd

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="einarfd's avatar" src="https://avatars.githubusercontent.com/u/5199247?s=50" width="50" height="50">](https://github.com/einarfd)<a name="note_1"></a>|@einarfd commented|
|-|-|

Merge request for bug at https://github.com/twisted/twisted/pull/393

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set status to `closed`|
|-|-|

In changeset 4f0baf8aa1c73f9dcde4f6acab67b57f57a4cae7
```
#!CommitTicketReference repository="" revision="4f0baf8aa1c73f9dcde4f6acab67b57f57a4cae7"
Merge pull request #394 from twisted/8654-einarfd-ssh-userauth-2

Author: einarfd
Reviewer: rodrigc
Fixes: #8654 

Port twisted.conch.ssh.userauth to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
